lib/src/coap_message.dart

```diff
@@ -148,8 +146,10 @@ class CoapMessage {
   typed.Uint8Buffer? get token => _token;
 
   /// As a string
-  String get tokenString =>
-      _token != null ? CoapByteArrayUtil.toHexString(_token!) : '';
+  String get tokenString {
+    final token = _token;
```

Comment:
What is the purpose of this variable?

Reply:
It is simply introduced for type promotion purposes, making it possible to get rid of the null check operator. This is not a must, as `_token` is obviously not `null` when the operator is used, but it is one potential source for bugs less in the future :)

Reply:
Ah, gotcha.